builder/cli-all-release.sh

```
@@ -22,6 +22,27 @@ function generatePlatformArchive() {
     popd || exit
 }
 
+function generateHomebrewBottles() {
+    TEMP_BOTTLE_DIRECTORY_NAME=chor
+    TEMP_BOTTLE_DIRECTORY=$TEMP_BUILD_DIRECTORY/$TEMP_BOTTLE_DIRECTORY_NAME
+
+    AGGREGATED_GO_LDFLAGS="$GO_LDFLAGS -X $PROJECT_MODULE/internal/pkg/build.buildPlatform=darwin/amd64"
+
+    GOOS=darwin GOARCH=amd64 go build -o "$TEMP_BOTTLE_DIRECTORY"/"$CHOREO_CLI_VERSION"/bin/chor \
+                            -ldflags "$AGGREGATED_GO_LDFLAGS" -x "$PROJECT_MODULE/$CHORE_CLI_SRC_ROOT"
+
+    cp "$PROJECT_ROOT"/LICENSE "$TEMP_BOTTLE_DIRECTORY"/"$CHOREO_CLI_VERSION"
+
+    pushd "$TEMP_BUILD_DIRECTORY" || exit
```

Shouldn't we return an error code when exiting?

---

fixed

---

fixed